docs/sources/reference/cli/run.md

```diff
@@ -55,6 +55,8 @@ The following flags are supported:
 * `--cluster.tls-cert-path`: Path to the certificate file used for peer communication over TLS.
 * `--cluster.tls-key-path`: Path to the key file used for peer communication over TLS.
 * `--cluster.tls-server-name`: Server name used for peer communication over TLS.
+* `--cluster.wait-for-size`: Wait for the cluster to reach the specified number of instances before admitting traffic to components that use clustering. Zero means disabled (default `0`).
```

Comment:
Is "admitting traffic" going to be clear language to users? Should it be more literal?
> Wait for the cluster to reach the specified number of instances before allowing components that use clustering to begin collecting telemetry...